src/hotspot/share/memory/memoryReserver.cpp

```diff
@@ -433,13 +433,13 @@ ReservedSpace HeapReserver::Instance::try_reserve_range(char *highest_start,
   const uint64_t num_attempts_to_try   = MIN2((uint64_t)HeapSearchSteps, num_attempts_possible);
```

Comment:
pre-existing, but github won't let me comment on line 429.  That pointer subtraction should be using
`pointer_delta(highest_start, lowest_start)`.

Reply:
Thanks, pointer substraction has been replaced as `pointer_delta`.